pkg/users/accounts-list.js

```diff
@@ -430,20 +416,13 @@ export const AccountsMain = ({ accountsInfo, current_user, groups }) => {
             return false;
         return true;
     });
-    const groupsExtraInfo = sortGroups(
-        groups.map(group => {
-            const userlistPrimary = accounts.filter(account => account.gid === group.gid).map(account => account.name);
-            const userlist = group.userlist.filter(el => el !== "");
-            return ({ ...group, userlistPrimary, userlist, members: userlist.length + userlistPrimary.length, isAdmin: admins.includes(group.name) });
-        })
-    );
 
     return (
         <Page id="accounts">
             <PageSection>
                 <Stack hasGutter>
-                    <GroupsList accounts={accounts} groups={groupsExtraInfo} />
-                    <AccountsList accounts={accounts} current_user={current_user} groups={groupsExtraInfo} />
+                    <GroupsList accounts={accounts} groups={groups} />
+                    <AccountsList accounts={accounts} current_user={current_user} groups={groups} />
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221212-195940-2c392d36-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)